tests/quantization/test_hawq_precision_init.py

```diff
@@ -288,8 +276,8 @@ def __str__(self):
 
 
 HAWQ_TEST_PARAMS = (
-    HAWQTestStruct(config_builder=HAWQConfigBuilder().pattern_based()),
-    HAWQTestStruct(config_builder=HAWQConfigBuilder().staged().pattern_based()),
+    HAWQTestStruct(config_builder=HAWQConfigBuilder()),
```

Comment:
no need to test with default arguments, it should be equal to the case with `for_cpu()`

Reply:
Done